File: api-examples/cat/search/python/attribute_search.py
import requests

url = "https://catalogue.iudx.org.in/catalogue/v1/search"

querystring = {"attribute-name":"%28tags%29","attribute-value":"%28traffic%29"}

payload = ""

response = requests.request("GET", url, data=payload, params=querystring)

print(response.text)

File: api-examples/cat/search/java/attribute_search.java
OkHttpClient client = new OkHttpClient();

Request request = new Request.Builder()
  .url("https://catalogue.iudx.org.in/catalogue/v1/search?attribute-name=%28tags%29&attribute-value=%28traffic%29")
  .get()
  .build();

Response response = client.newCall(request).execute();